autoload/neomake/makers/ft/sml.vim

```diff
@@ -0,0 +1,29 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#sml#EnabledMakers()
+    if exists('s:sml_makers')
+        return s:sml_makers
+    endif
+
+    let makers = []
+    if neomake#utils#Exists('sml')
+        call add(makers, 'smlnj')
+    endif
+
+    let s:sml_makers = makers
+    return makers
+endfunction
+
+" This comes straight out of syntastic.
+function! neomake#makers#ft#sml#smlnj
```

Comment:
There are many implementations of standard ml, the sml command is only provided by the SML/NJ implementation(which usually has a package name of `smlnj`).